lib/scr/screen_dice_roller.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:simple_dice_roller/utils/constants_file.dart';
```

Comment:
Change for a relative import here, please

Reply:
Done